src/field_10x26_impl.h

```diff
@@ -10,6 +10,10 @@
 #include "util.h"
 #include "field.h"
 
+#if defined(VALGRIND)
```

Comment:
I think for now it's cleaner  to check also `VERIFY` here. (Same in the other files)